<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddKeyDialog</class>
 <widget class="QDialog" name="AddKeyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>131</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>293</width>
    <height>131</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>293</width>
    <height>131</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add Key</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,5">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Key Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="keyNameInput"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,5">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Key Type:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="keyTypeList">
         <item>
          <property name="text">
           <string notr="true">string</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">hash</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">list</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">set</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">zset</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>stream</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddKeyDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddKeyDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
